doc/src/postgresql.rst

```diff
@@ -8,7 +8,7 @@ Managed instance of the `PostgreSQL <http://postgresql.org>`_ database server.
 Components
 ----------
 
-* PostgreSQL server (9.6, 10, 11, or 12)
+* PostgreSQL server (9.5, 9.6, 10, 11, or 12)
```

Comment:
is 9.6 really available? We also have 13 now

Reply:
9.5 and 9.6 are available